pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ApexLanguageModule.java

```diff
@@ -6,21 +6,31 @@
 
 import static net.sourceforge.pmd.util.CollectionUtil.listOf;
 
+import java.util.List;
+
+import net.sourceforge.pmd.annotation.InternalApi;
 import net.sourceforge.pmd.lang.BaseLanguageModule;
 import net.sourceforge.pmd.lang.Language;
 import net.sourceforge.pmd.lang.LanguageRegistry;
 import net.sourceforge.pmd.lang.apex.internal.ApexHandler;
 
-import apex.jorje.services.Version;
-
 public class ApexLanguageModule extends BaseLanguageModule {
 
     public static final String NAME = "Apex";
     public static final String TERSE_NAME = "apex";
 
+    @InternalApi
+    public static final List<String> VERSIONS = listOf("52", "53", "54", "55", "56", "57");
```

Comment:
I've removed the `VERSIONS` field. However, we can't use `ApexLanguageModule.getInstance()` here, because we are in the middle of initializing the language registry (so `LanguageRegistry.PMD` is still null).
I'm now creating a new temporary apex module in VF in order to retrieve the versions.